apm-agent-plugins/apm-jaxrs-plugin/src/main/java/co/elastic/apm/agent/jaxrs/JaxRsTransactionNameInstrumentation.java

```diff
@@ -54,18 +57,25 @@
 
     private final Collection<String> applicationPackages;
     private final JaxRsConfiguration configuration;
+    @Nullable
+    @VisibleForAdvice
+    public static JaxRsTransactionHelper jaxRsTransactionHelper;
+    private final ElasticApmTracer elasticApmTracer;
```

Comment:
No need- the parent abstract instrumentation already holds a reference to the tracer

Reply:
this object we pass to the JaxRsOffsetMappingFactory

Reply:
You pass a reference, you can pass the `tracer` reference from the parent, can't you?

Reply:
I passed it from
```
private static AgentBuilder.Transformer.ForAdvice getTransformer(final ElasticApmTracer tracer, final ElasticApmInstrumentation instrumentation, final Logger logger, final ElementMatcher<? super MethodDescription> methodMatcher) {
        Advice.WithCustomMapping withCustomMapping = Advice
            .withCustomMapping()
            .bind(new SimpleMethodSignatureOffsetMappingFactory())
            .bind(new AnnotationValueOffsetMappingFactory());
        Advice.OffsetMapping.Factory<?> offsetMapping = instrumentation.getOffsetMaping(tracer);
    
```
is it correct?